doc/source/user/absolute_beginners.rst

```diff
@@ -967,9 +967,8 @@ All you need to do is pass in the number of elements you want it to generate::
   array([1., 1., 1.])
   >>> np.zeros(3)
   array([0., 0., 0.])
-  # the simplest way to generate random numbers
-  >>> rng = np.random.default_rng(0)
-  >>> rng.random(3)
+  >>> rng = np.random.default_rng(0) # the simplest way to generate random numbers
```

Comment:
can you remove the seed here?

and small PEP8 thing: 2 spaces before `#`

Reply:
done by reverting, see above.